modules/persistent-storage-cinder-volume-security.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent-storage-cinder.adoc
+
+[id="persistent-storage-cinder-volume-security_{context}"]
+= Cinder volume security
+
+If you use Cinder PVs in your application, configure security for their
+deployment configurations.
+
+. Create an SCC that uses the appropriate `*fsGroup*` strategy.
```

Comment:
If there aren't instructions on how to accomplish this step we could move it to a Prerequisite.

Reply:
Moved to Prereq as suggested.